app/controllers/coffee_controller.rb

```diff
@@ -1,11 +1,14 @@
 class CoffeeController < ApplicationController
   
   def index
-    @most_recent_brew = Brew.last
+    if current_user 
+      @most_recent_brew = Brew.where(team_id: current_user.team_id).last
```

Comment:
how about this
if current_user
  @most_recent_brew = Brew.get_most_recent_brew
else
  @most_recent_brew = # return fake brew object saying to start brewing coffee
end